corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -266,6 +274,114 @@ public int dropTable(String namespace, String tablename) {
         return tableSize;
     }
 
+    public int editRecord(String namespace, String tableName,
+        String keyToEdit, String newRecord) {
+        verifyNamespaceAndTablename(namespace, tableName);
+
```

Comment:
Can we have a verify function to check null value and empty?
like verifyInput(String str, String inputName)
and all null or empty value will see the consistent IllegalArugumentException

Reply:
cleaned the verifyNamespace and verifyNamespaceAndTablename methods and unified them.  But I think it is valuable to print which param is null or empty